tests/testing.h

```diff
@@ -36,7 +36,7 @@
 #include <sstream>
 #include <functional>
 
-#ifdef _WIN32
+#if (defined _WIN32) || (defined SDL2PP_XCODE_GENERATED)
```

Comment:
So that mean we should just go ahead and use catch? I could add it for the event handler tests (issue #51 for ref)